src/components/protocol_handler/src/protocol_packet.cc

```diff
@@ -47,6 +48,27 @@ namespace protocol_handler {
 
 CREATE_LOGGERPTR_GLOBAL(logger_, "ProtocolHandler")
 
+namespace {
+std::string StringifyFrameType(uint8_t type) {
+  switch (type) {
+    case FRAME_TYPE_SINGLE:
+      return "FRAME_TYPE_SINGLE";
+
+    case FRAME_TYPE_CONSECUTIVE:
+      return "FRAME_TYPE_CONSECUTIVE";
+
+    case FRAME_TYPE_CONTROL:
```

Comment:
@KVGrygoriev just for case I would like to add `FRAME_TYPE_FIRST` as well

Reply:
@AKalinich-Luxoft see e9e7728e3c0b11328d55b083a8bfcbd1542c1f89